### ISSUE TITLE
get masks

### ISSUE THREAD

**Issue by widedh** [mhamilton723/STEGO#81]
Hello everyone,

How can I get the mask of output images, please? thank you

**Comment by tanveer6715** [comment]
Yes you can get mask by changing model path in eval config file to your save checkpoint file and then run python eval_segmentation.py file.

**Comment by www545** [comment]
Error executing job with overrides: []
Traceback (most recent call last):
  File "G:\STEGO\src\eval_segmentation.py", line 187, in my_app
    plot_img = (prep_for_plot(saved_data["img"][img_num]) * 255).numpy().astype(np.uint8)
IndexError: index 260 is out of bounds for dimension 0 with size 260
how to get all the mask of output images ?
